src/components/AssetsComponent.jsx

```diff
@@ -29,28 +34,45 @@ export default class AssetsComponent extends Component {
   }
 
   shouldComponentUpdate(nextProps) {

```

Comment:
Do we really need this here? I think it is safe to delete `shouldComponentUpdate` as it was initially added for the toast message, which we are shifting away from